src/index.html

```diff
@@ -10,19 +10,25 @@
         <div id="the-game"></div>
         <div class="settings">
             <nav class="mobile-nav">
-                navigation and menu link
```

Comment:
I am thinking of keeping here informations like health/energy points (probably in a form of a bars) so it won't be just a button. But it can be removed for now

Reply:
Hmm, then it should not be visible for mobile only, we probably need some another fixed overlay for this.

Reply:
yup, but I don't know yet if it's better to keep it in the map area or on the right panel (on desktop)
we can discuss it :)